examples/Forms/CheckLabel.md

```diff
@@ -0,0 +1,121 @@
+The `CheckLabel` component is used by the `RadioButton` and `Checkbox` components, while the `InputLabel` component is used by the `TextInput` and `Dropdown` components. These label variations were created to align input fields when using various form elements in tandem.
+
+Aligning `RadioButton` and `Dropdown` fields example: The labelPosition for `RadioButton` is `POSITION.RIGHT`, and the `labelPosition` for the `Dropdown` is `POSITION.TOP`.
+```jsx
+import { useState } from 'react';
+import { Fieldset, RadioButton, Dropdown, POSITION } from 'mark-one';
+const Example = () => {
```

Comment:
For all of these examples, you can call `useState` directly -- you don't need to wrap everything in the `<Example>` component. It's not necessary wrong, but we should try to keep our example code as clean as possible.

Reply:
That's a great suggestion! I found that a few other Form element examples were also wrapping in an Example component, so I updated those as well through 1f1c998. Definitely a lot cleaner!!